apps/homework/tasks.py

```diff
@@ -33,37 +34,48 @@ def post_submission(submission_pk):
     )
 
     # Example of url format we're expecting: competition/15595/submission/44798/4aba772a-a6c1-4e6f-a82b-fb9d23193cb6.zip
-
+    if not resp.ok:
+        return
```

Comment:
should we raise an error of some kind or set a status or anything on a fail?

Reply:
👍